MyLibrary/Library/Controllers/BookController.cs

```diff
@@ -0,0 +1,157 @@
+﻿// <copyright file="BookController.cs" company="Peretiatko Anastasiia">
+// Copyright (c) Peretiatko Anastasiia. All rights reserved.
+// </copyright>
+
+using System;
+using BusinessLogic.LibraryModel;
+using BusinessLogic.LibraryService;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Library.Controllers
+{
+    /// <summary>
+    /// Controller for dealing with Book
+    /// </summary>
+    /// <seealso cref="Microsoft.AspNetCore.Mvc.ControllerBase" />
+    [Route("api/[controller]")]
+    [ApiController]
+    public class BookController : ControllerBase
+    {
+        /// <summary>
+        /// The library service
+        /// </summary>
+        private ILibraryService libraryService;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="BookController"/> class
+        /// </summary>
+        /// <param name="libraryService">The library service</param>
+        public BookController(ILibraryService libraryService)
+        {
+            this.libraryService = libraryService;
+        }
+
+        // GET: api/book
+        /// <summary>
+        /// Gets all books from the Library
+        /// </summary>
+        /// <returns>
+        /// HTTP status code Ok() with the list of books
+        /// or HTTP status code NoContent() if there are any books in library
+        /// </returns>
+        [HttpGet]
+        public IActionResult GetAllBooks()
+        {
+            if (this.libraryService.GetAllBooks() != null)
+            {
+                return Ok(this.libraryService.GetAllBooks());
+            }
+
+            return NoContent();
+        }
+
+        // GET: api/book/5
+        /// <summary>
+        /// Gets the book by its id
+        /// </summary>
+        /// <param name="id">The identifier of the book</param>
+        /// <returns>
+        /// HTTP status code Ok() with found book
+        /// or HTTP status code NotFound() if there isn`t book with such id
+        /// </returns>
+        [HttpGet("{id}")]
+        public IActionResult GetBook(uint id)
+        {
+            Book foundBook = this.libraryService.GetBookById(id);
+            if (foundBook == null)
+            {
+                return NotFound("No books with such id!");
+            }
+
+            return Ok(foundBook);
+        }
+
+        // POST: api/book
+        /// <summary>
+        /// Adds the new book to the book list
+        /// </summary>
+        /// <param name="book">The book for adding</param>
+        /// <returns>
+        /// HTTP status code Created() with added book`s id
+        /// or HTTP status code BadRequest() with error message
+        /// </returns>
+        [HttpPost]
+        public IActionResult AddNewBook([FromBody]Book book)
+        {
+            // ???
```

Comment:
https://stackoverflow.com/questions/38962288/modelstate-isvalid-always-true-regardless-of-dataannotations-attributes maybe that's will help you